data-migrator/core/src/main/java/io/camunda/migration/data/config/InterceptorConfiguration.java

```diff
@@ -83,7 +83,7 @@ public List<VariableInterceptor> configuredVariableInterceptors() {
    */
   @Bean
   public List<EntityInterceptor> configuredEntityInterceptors() {
-    ConfigurationLogs.logConfiguringInterceptors();
+    ConfigurationLogs.logConfiguringInterceptors("entity");
```

Comment:
Nit: could introduce a constant for "entity"

Reply:
I will move it to a follow up ticket